src/lib/components/billing/estimatedTotal.svelte

```diff
@@ -0,0 +1,146 @@
+                message: e.message
+            });
+        }
+    }
+
+    async function getUpdatePlanEstimate(
+        organizationId: string,
+        billingPlan: string,
+        collaborators: string[],
+        couponId: string | undefined
+    ) {
+        try {
+            estimation = await sdk.forConsole.billing.estimationUpdatePlan(
+                organizationId,
+                billingPlan,
+                couponId && couponId.length > 0 ? couponId : null,
+                collaborators ?? []
+            );
+        } catch (e) {
+            if (e instanceof AppwriteException) {
+                if (
+                    e.type === 'coupon_not_found' ||
+                    e.type === 'coupon_already_used' ||
+                    e.type === 'billing_credits_unsupported'
+                ) {
+                    couponData = {
+                        code: null,
+                        status: null,
+                        credits: null
+                    };
+                }
+            }
+            addNotification({
+                type: 'error',
+                isHtml: false,
+                message: e.message
+            });
+        }
+    }
+
+    $: organizationId
+        ? getUpdatePlanEstimate(organizationId, billingPlan, collaborators, couponData?.code)
+        : getEstimate(billingPlan, collaborators, couponData?.code);
+</script>
+
+{#if estimation}
+    <Card class="u-flex u-flex-vertical u-gap-8">
+        <slot />
+        {#if estimation}
+            {#each estimation.items ?? [] as item}
+                {#if item.value > 0}
+                    <span class="u-flex u-main-space-between">
+                        <p class="text">{item.label}</p>
+                        <p class="text">{formatCurrency(item.value)}</p>
+                    </span>
+                {/if}
+            {/each}
+            {#each estimation.discounts ?? [] as item}
+                <DiscountsApplied bind:couponData {...item} />
+            {/each}
+            <div class="u-sep-block-start" />
+            <span class="u-flex u-main-space-between">
+                <p class="text">Total due</p>
+                <p class="text">
+                    {formatCurrency(estimation.grossAmount)}
+                </p>
+            </span>
+
+            <p class="text u-margin-block-start-16">
+                You'll pay <span class="u-bold">{formatCurrency(estimation.grossAmount)}</span> now.
+                Once your credits run out, you'll be charged
+                <span class="u-bold">{formatCurrency(estimation.amount)}</span> every 30 days.
+            </p>
+        {/if}
+
+        <FormList class="u-margin-block-start-24">
+            <InputChoice
+                type="switchbox"
+                id="budget"
+                label="Enable budget cap"
+                tooltip="If enabled, you will be notified when your spending reaches 75% of the set cap. Update cap alerts in your organization settings."
```

Comment:
isn't this 80%?

Reply:
For org with budgets (non-free) if they don't update, default is 75%
https://github.com/appwrite-labs/cloud/blob/ef81f8887b39c686618e32683c2675aa48127505/src/Appwrite/Cloud/Platform/Modules/Organizations/Http/Organizations/UpdateBudget.php#L45